sniffles/rulereader.py

```diff
@@ -1125,10 +1141,6 @@ def setHttpEncode(self, h=None):
 
 class SnortRuleParser(RuleParser):
 
-    def __init__(self, filename=None):
```

Comment:
Yes, this was causing problem because in readRuleFile, parser variable can be a instance of SnortRuleParser and PetabiRuleParser. So when readRuleFile was trying to retrieve information about backgroundTraffic (using getBackgroundTraffic() method in parent class RuleParser) from SnortRuleParser (which does not have self.backgroundTraffic variable in its __init__), an error was occurring in Sniffles.